docs/sidebars.js

```diff
@@ -69,9 +69,7 @@ const sidebars = {
     {
       type: "category",
       label: "Tutorials",
-      items: [
-        "guides/tutorials/php",
-      ],
+      items: ["guides/tutorials/php", "guides/tutorials/userId"],
```

Comment:
```suggestion
      items: ["guides/tutorials/php", "guides/tutorials/userData"],
```

And rename the file to `userData.mdx`.